tests/test_ser.py

```diff
@@ -95,6 +95,14 @@ def fail(x):
         s.serialize_value(Q())
 
 
+def test_subclass():
+    class SubclassedUUID(uuid.UUID):  # such as asyncpg.pgproto.pgproto.UUID
+        pass
+
+    _uuid = s.serialize_value(SubclassedUUID("939d4cc9-830d-4cca-bd74-3ec3d541a9b3"))
+    assert _uuid == "uuid:939d4cc9-830d-4cca-bd74-3ec3d541a9b3"
+
+
```

Comment:
Could you please add a test that ensures mro is traversed in the correct order if multiple classes in an inheritance heirarchy have custom serializers defined?

Reply:
done!